src/js/components/DataTable/doc.js

```diff
@@ -452,6 +481,12 @@ export const themeDoc = {
     type: 'React.Element',
     defaultValue: undefined,
   },
+  'dataTable.pagination': {
+    description: `If paginating, any valid Box props to style the pagination 
+    component.`,
+    type: 'object',
+    defaultValue: `margin: { top: 'xsmall' }`,
```

Comment:
I don't see this default value in base.js, I think we achieve the desired styling with dataTable.container. Can this be removed?

Reply:
Fixed.